test/util/appendingDirectOnly.js

```diff
@@ -0,0 +1,185 @@
+module.exports = fallbackDependancySandBox
+
+function fallbackDependancySandBox (appDir) {
+  const fs = require('fs')
+  const path = require('path')
+  const testSrc = path.resolve(__dirname, '../../test')
+  const { execSync } = require('child_process')
+  const repoList = ['repo1', 'repo2', 'repo3']
+
+  if (!appDir) {
+    let processEnv
```

Comment:
likewise processEnv is never used for anything. i think we can remove all references to this too